### DISCUSSION TITLE
挑戰地底救災，美國防部在巨大礦坑內舉辦機器人比賽

### DISCUSSION THREAD

**Discussion by 239216** [1101-GS4719-C/Python-Community-GS4719-C#84] [General]
以現今的技術，無人機或機器人在開闊地形，訊號傳遞良好的位置固然能夠順利運作，但許多情況下，無論戰場或是救災都位於倒塌建築，或地下通道這種沒有地圖跟無線電訊號的艱困場所，戰鬥或救災人員進入這類場地執行任務都相當危險。

國防高等研究計劃署（DARPA）舉辦了機器人「地底挑戰賽」，以高額獎金吸引世界各地參賽團隊，使用現有的機器人或小型無人機平台改裝，加入機器視覺（Machine Vision）及機器學習（Machine Learning）等人工智慧能力，讓機器人以 3~4 機一組，進入地底艱困場地，探索現場繪製地圖，並找出 DARPA 預先放置的「生還者」，以及各種指定物品。
https://technews.tw/2021/10/07/darpas-sub-challenge-won-by-team-cerberus-and-got-2-million-dollar-prize/

**Comment by enoch-lightning** [comment]
這確實是一項有意義的競賽。開發機器人，就是為幫助人類做到不容易，或是不可能做到的事。

**Comment by chupochun** [comment]
這個競賽感覺方長有趣 若是以後使用在救災上肯定會擁有很大的幫助

**Comment by pekoranoinu** [comment]
開發這種機器人能夠在救災上有很大的幫助，和人類互補，看來這會成為不錯的科技開發走向。

**Comment by xkllkx** [comment]
希望研發成功後台灣可以引進，能夠減少救災人員的傷亡

**Comment by boson13579** [comment]
!!這種比賽才是我們需要的比賽吧!，現在一堆ai的比賽都在比麼預測的準確率，雖然不能說沒用，但是比起這種能真的改變人們生活，或是對社會有貢獻的比賽，真的是小巫見大巫了，希望台灣(或是瘸世界)的AI比賽都可以朝這種方向努力!
